os/dbuild.sh

```diff
@@ -140,6 +143,15 @@ function SELECT_OPTION()
 		6|smartfs)
 			BUILD smartfs
 			;;
+		7|trap)
+			if [ "${STATUS}" == "BUILT" ]; then
+				echo "Please provide name of log file created in tools/trap:"
```

Comment:
How about taking the file path relative to root/tizenrt/os? The user again needs to create the file in a different directory and run the command from os according to the current implementation